Example/RxGesture-OSX/ViewController.swift

```diff
@@ -28,11 +28,22 @@ class Step {
 
 class MacViewController: NSViewController {
 
+    #if compiler(>=5.2)
+    /// Typed parent object.
+    ///
+    /// - note: Since Swift 5.2 and Xcode 11.4, Apple have suddenly
+    ///         disallowed using `weak` for NSTextView. For more details
+    ///         see this GitHub Issue: https://git.io/JvSRn
+    @IBOutlet private var code: NSTextView!
+    #else
+    /// Typed parent object.
+    @IBOutlet private weak var code: NSTextView!
+    #endif
+
```

Comment:
@Loupehope do we really need to have this conditional code? Can't we have a strong reference on < 5.2 too? What could go wrong?

Reply:
@jegnux looks like there is no problem with strong and @IBOutlet for NSTextView -> remove check.

Reply:
Thanks!